src/main/resources/templates/ftl-bits/ext-css.ftl

```diff
@@ -1,7 +1,17 @@
+<!-- Fonts -->
+<link rel="stylesheet" type="text/css" href='http://fonts.googleapis.com/css?family=Roboto+Condensed:300,400'>
+<link rel="stylesheet" type="text/css" href='http://fonts.googleapis.com/css?family=Lato:300,400,700,900'>
 <!-- Bootstrap -->
-<link href="webjars/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="webjars/bootstrap/3.3.6/css/bootstrap.min.css">
 <!-- Libraries -->
-<link href="webjars/ng-tags-input/3.0.0/build/ng-tags-input.min.css" rel="stylesheet">
-<link href="webjars/angularjs-toaster/0.4.15/toaster.min.css" rel="stylesheet">
-<link href="webjars/angular-xeditable/0.1.9/dist/css/xeditable.css" rel="stylesheet">
-<link href="webjars/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="webjars/ng-tags-input/3.0.0/build/ng-tags-input.min.css">
+<link rel="stylesheet" type="text/css" href="webjars/angularjs-toaster/0.4.15/toaster.min.css">
+<link rel="stylesheet" type="text/css" href="webjars/angular-xeditable/0.1.9/dist/css/xeditable.css">
+<link rel="stylesheet" type="text/css" href="webjars/font-awesome/4.5.0/css/font-awesome.min.css">
+<!-- CSS Libs -->
+<link rel="stylesheet" type="text/css" href="webjars/animate.css/3.5.0/animate.css">
+<link rel="stylesheet" type="text/css" href="webjars/bootstrap-switch/3.3.2/dist/css/bootstrap3/bootstrap-switch.min.css">
+<link rel="stylesheet" type="text/css" href="webjars/checkbox3/0.1.1/dist/checkbox3.min.css">
+<link rel="stylesheet" type="text/css" href="webjars/DataTables/1.10.10/media/css/jquery.dataTables.min.css">
```

Comment:
Similar comment about trimming down theme css that aren't needed. For data tables I will use [ui-grid](http://ui-grid.info/).